app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/addToTransforms.ts

```diff
@@ -26,3 +25,8 @@ export const addRenameToTransforms = (transform: IColumnTransformation): string
   const { columnName, directive } = transform;
   return `rename ${columnName} ${directive}`;
 };
+
+export const addMaskToTransforms = (transform: IColumnTransformation): string => {
+  const { columnName, directive } = transform;
+  return `mask ${columnName} ${directive}`;
```

Comment:
Should column names be preceded by `:`?

Reply:
@njbriggs afaik this is how wrangler does them `directiveName columnName directiveText` just in a string - it works for rename that way tested all the way through.

Reply:
We'd like to consistently use `:` for column names in Wrangler: https://cdap.atlassian.net/browse/CDAP-17765

Reply:
I'd have to look at the code for the plugin and do some testing to validate that works. According to this https://docs.google.com/document/d/17yJjaLs2czElogB9RizTqPIGPB8kcIU9vNjwXfVza3I/edit?resourcekey=0-wdGszG0eqOtnLyBPSHwQhA#bookmark=id.yrlg98x0gyh2 its supposed to be without colons.

Reply:
@njbriggs can we get this code in and then i can do another pr addressing that if my testing etc works. It works without the `:` for now

Reply:
OK then, let's get this merged.

Reply:
cool - made this jira task to keep track of it https://cdap.atlassian.net/browse/CDAP-18698